### PR TITLE
Fix issue in reboot

### PR DESCRIPTION
**Pull request by xutian**
iD: 1186124

Signed-off-by: Xu Tian <xutian@redhat.com>

### CODE REVIEW

File: virttest/qemu_vm.py [hunk 1]
```diff
@@ -3493,31 +3493,38 @@ def reboot(self, session=None, method="shell", nic_index=0,
...
-        def __rebooting(session):
+        def __rebooting(session, timeout):
             """
             Check guest rebooting.
             """
             if self.params["os_type"] == "windows":
                 timeout = self.CLOSE_SESSION_TIMEOUT
                 return not session.is_responsive(timeout=timeout)
-            return session.read_until_any_line_matches("Rebooting")
+            return session.read_until_any_line_matches("Rebooting",
```
Comment (ypu) [inline virttest/qemu_vm.py]: patterns here should be an list but not a string.

**Comment by xutian** [comment]
@ypu 

Thanks for your comment, pullreq has updated.


Thanks,
@xutian

**Comment by xiagao** [comment]
Acked-by:Xiaoling Gao xiagao@redhat.com

**Comment by ypu** [comment]
LGTM

Acked-by: Yiqiao Pu <ypu@redhat.com>

**Comment by ypu** [comment]
Thanks @xutian and @xiagao will merge it.

**Pull request merged**